x/upgrade/README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Abstract
 
-The upgrade module removes the entrypoints to the standard upgrade module by not
-registering a message server. It registers the standard upgrade module types to
-preserve the ability to marshal them. Note that the keeper of the standard
-upgrade module is still added to the application.
+This upgrade module is a fork of the cosmos-sdk's
+[x/upgrade](https://github.com/cosmos/cosmos-sdk/tree/main/x/upgrade) module
+that removes the entrypoints to the standard upgrade module by not registering a
+message server. It registers the standard upgrade module types to preserve the
+ability to marshal them. Note that the keeper of the standard upgrade module is
+still added to the application.
+
+A consequence of the removal of the entrypoints is that
+[cosmosvisor](https://github.com/cosmos/cosmos-sdk/tree/main/tools/cosmovisor)
+will not work for celestia-app.
```

Comment:
this is mostly true, but I feel like it might be useful to expand on this point somehow. We could still use it if we needed to. For example, we can still save a json with the relevant info or log things in the same way if we ever needed to have the same type of upgrade. Neutering the upgrade module as we've done here doesn't mean we will never be able to use cosmovisor, simply that we cannot vote on an upgrade with tokens.

Perhaps we should either merge https://github.com/celestiaorg/celestia-app/pull/1562 or I can maybe move portions of it here as well.

Reply:
Removed the note on cosmovisor https://github.com/celestiaorg/celestia-app/pull/2072/commits/2d0503e33b9950937ddfe67f4f1b30f49faf3145

Reply:
thanks!